docs/user_guide/assessment.rst

```diff
@@ -1,20 +1,109 @@
-Assessment
-==========
+Introduction
+============
 
-Metrics
--------
+The goal of fairness assessment is to answer the question: Which groups of 
+people may be disproportionately negatively impacted by an AI system and in 
+what ways?
 
-.. currentmodule:: fairlearn.metrics
+The steps of the assesment are as follows:
+1. Identify harms
+2. Identify the groups that might be harmed
+3. Quantify harms
+4. Compare quantified harms across the groups
+
+We next examine these four steps in more detail.
+
+1. Identify harms
+-----------------
+
+For example, in a system for screening job applications, qualified candidates 
+that are automatically rejected experience an allocation harm. In a 
+speech-to-text transcription system, high error rates constitute harm in the 
+quality of service.
+
+2. Identify the groups that might be harmed
+-------------------------------------------
+
+In most applications, we consider demographic groups including historically 
+marginalized groups (e.g., based on gender, race, ethnicity). We should also 
+consider groups that are relevant to a particular application. For example, for 
+speech-to-text transcription, groups based on the regional dialect or being a  
+native or a non-native speaker.
+
+It is also important to consider group intersections, for example, in addition
+to considering groups according to gender and groups according to race, it is 
+also important to consider their intersections (e.g., Black women, Latinx 
+nonbinary people, etc.).
+
+3. Quantify harms
+-----------------
+
+Define metrics that quantify harms or benefits:
+
+* In job screening scenario, we need to quantify the number of candidates that are classified as "negative" (not recommended for the job), but whose true label is "positive" (they are qualified). One possible metric is the false negative rate: fraction of qualified candidates that are screened out.
+
+* In speech-to-text scenario, the harm could be measured by word error rate, number of mistakes in a transcript divided by the overall number of words.
+
+Note that in some cases, harms might not be readily identifiable. If you 
+choose to use a proxy to represent the harm, check the proxy regularly 
```

Comment:
So, we don't really explain proxies in our docs, other than briefly in the Boston housing dataset section and in the tutorial (which people reading this might not have read). 

Is this something we want to include here? If so, I think we should expand this to say more about what a proxy is in this case, and when and how people might want to identify proxies for sensitive features in their data (or not).

Reply:
I am fine with leaving it in or taking it out! I added a paragraph explaining what a proxy might look like in this case. If this isn't clear enough, I am fine with scrapping it. I've included the paragraph below, please let me know your thoughts.

> Note that in some cases, harms might not be readily identifiable. In these cases, we might choose to use a closely related variable to stand in for the missing variable. For example, suppose that in the job screening scenario, we have data on whether the candidate passes the first two stages, but not if they are ultimately recommended for the job. An alternative measure of harm might be if the candidate does not pass the first stage of the screen. If you choose to use a proxy variable to represent the harm, check the proxy variable regularly to ensure it remains useful over time.

Reply:
Suggesting further edits to highlight the definition and role of a proxy in this setting:

```
Note that in some cases, harms might not be readily quantifiable. 
In these cases, we might choose to use a closely related variable,
often called a "proxy",  
to stand in for the missing variable. For example, suppose that in 
the job screening scenario, we have data on whether the candidate passes 
the first two stages, but not if they are ultimately recommended for the 
job.
As an alternative to the unobservable final recommendation, we could
therefore measure the harm using the proxy variable indicating whether
the candidate passes the first stage of the screen.
If you choose to use a proxy variable to 
represent the harm, check the proxy variable regularly to ensure it 
remains useful over time.
```